CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/CameraXBasic.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.android.example.cameraxbasic
+
+import android.app.Application
+import androidx.camera.camera2.Camera2Config
+import androidx.camera.core.CameraXConfig
+
+/** It implements CameraXConfig.Provider */
+class CameraXBasic: Application(), CameraXConfig.Provider {
```

Comment:
You don't need to make an Application class that extends `CameraXConfig.Provider` for `alpha10`. That was a change between alpha 9 and 10.